modules/EnsEMBL/Web/Component/Gene/ComparaOrthologs.pm

```diff
@@ -0,0 +1,110 @@
+=head1 LICENSE
+
+Copyright [2009-2024] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package EnsEMBL::Web::Component::Gene::ComparaOrthologs;
+
+use strict;
+use warnings;
+
+our $GENE_TREE_CONSTANTS = {
+  default => {
+    name => 'Metazoa',
+    url_part => 'Compara_Tree'
+  },
+  protostomes => {
+    name => 'Protostomes',
+    url_part => 'Protostomes_Tree'
+  },
+  insects => {
+    name => 'Insects',
+    url_part => 'Insects_Tree'
+  },
+  pangenome_drosophila => {
+    name => 'Drosophilidae',
+    url_part => 'Drosophilidae_Tree'
+  }
+};
+
+our $METAZOA_CLUSTERSETS = [keys %$GENE_TREE_CONSTANTS];
```

Comment:
Could you find it in your heart to keep this as a hard-coded list, if I promise to submit a PR to remove it in time for `release/113`? 🥺

It could be useful in the meantime, to ensure that gene-tree links would be in a consistent order:
```suggestion
our $METAZOA_CLUSTERSETS = ['default', 'protostomes', 'insects', 'pangenome_drosophila'];
```